clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -1503,11 +1471,6 @@ int main(int argc, const char **argv) {
     return 0;
   }
 
-  // These calls are needed so that we can read bitcode correctly.
-  InitializeAllTargetInfos();
-  InitializeAllTargetMCs();
-  InitializeAllAsmParsers();
```

Comment:
It probably also makes sense to remove redundant link components from CMakeLists.txt (please see https://github.com/intel/llvm/pull/2886 for details).